assets/js/modules/ads/components/common/AdsConversionIDTextField.js

```diff
@@ -85,6 +85,11 @@ export default function AdsConversionIDTextField( { helperText } ) {
 						'google-site-kit'
 					)
 				}
+				leadingIcon={
+					<span className="googlesitekit-text-field-conversion-tracking-id-prefix">
+						{ __( 'AW-', 'google-site-kit' ) }
```

Comment:
Should this really be translated?

```suggestion
						{ __( 'AW-', 'google-site-kit' ) }
```

I'm not sure the value of it, maybe using `_x()` would be better? 🤔

Reply:
Good point, it shouldn't change based on the language so how about:

```suggestion
						AW-
```

Reply:
My only concern with not localising it is that it'll be seen as a bug/there could possibly be a change for certain locales I can't predict. `_x` is probably the right call, since it indicates it probably shouldn't be changed but leaves the option? 🤔

Reply:
For now, I'll just leave a comment explaining it, at least 😄